client/app/deskaas/deskaas-change-password/deskaas-change-password.html

```diff
@@ -1,50 +1,41 @@
-            <div data-ng-if="!vm.flags.init">
-                <div>
-                    <ul>
-                        <li data-translate="vdi_pwd_policy_length" data-translate-values="{min : vm.policies.minLength, max : vm.policies.maxLength}"></li>
-                        <li data-translate="vdi_pwd_policy_letterMandatory" data-ng-if="vm.policies.letterMandatory"></li>
-                        <li data-translate="vdi_pwd_policy_lowerMandatory" data-ng-if="vm.policies.lowercaseLetterMandatory"></li>
-                        <li data-translate="vdi_pwd_policy_upperMandatory" data-ng-if="vm.policies.uppercaseLetterMandatory"></li>
-                        <li data-translate="vdi_pwd_policy_digitMandatory" data-ng-if="vm.policies.digitMandatory"></li>
-                        <li data-translate="vdi_pwd_policy_specialMandatory" data-ng-if="vm.policies.specialMandatory"></li>
-                        <li data-translate="vdi_pwd_policy_deniedChars" data-translate-values="{chars : vm.policies.deniedCharsString}"></li>
-                    </ul>
+<form novalidate name="form" class="deskaas-change-password" data-ng-submit="(form.$valid || vm.values.generatePwd) && vm.ok()">
+    <oui-modal data-title="{{ ::'vdi_pwd_title' | translate }}"
+               data-primary-label="{{ ::'common_modal_ok' | translate }}"
+               data-secondary-action="vm.cancel()"
+               data-secondary-label="{{ ::'common_modal_cancel' | translate }}"
+               data-on-dismiss="vm.cancel()"
+               data-loading="vm.flags.init">
+        <div data-ng-if="!vm.flags.init">
+            <ul>
+                <li data-translate="vdi_pwd_policy_length" data-translate-values="{min : vm.policies.minLength, max : vm.policies.maxLength}"></li>
+                <li data-translate="vdi_pwd_policy_letterMandatory" data-ng-if="vm.policies.letterMandatory"></li>
+                <li data-translate="vdi_pwd_policy_lowerMandatory" data-ng-if="vm.policies.lowercaseLetterMandatory"></li>
+                <li data-translate="vdi_pwd_policy_upperMandatory" data-ng-if="vm.policies.uppercaseLetterMandatory"></li>
+                <li data-translate="vdi_pwd_policy_digitMandatory" data-ng-if="vm.policies.digitMandatory"></li>
+                <li data-translate="vdi_pwd_policy_specialMandatory" data-ng-if="vm.policies.specialMandatory"></li>
+                <li data-translate="vdi_pwd_policy_deniedChars" data-translate-values="{chars : vm.policies.deniedCharsString}"></li>
+            </ul>
+            <div class="oui-field__content">
+                <div class="oui-field-control oui-field-control_3">
+                    <input type="text" id="vdi-new-password"
+                        name="password"
+                        class="oui-input"
+                        data-ng-class="{ 'oui-input_error': form.$submitted && form.password.$invalid }"
+                        data-ng-model="vm.values.password"
+                        cui-password
```

Comment:
cui-password should become a ui-kit component, no?

Reply:
We dont have a similar component for password in ui-kit yet. Will migrate it once its released in ui-kit